### DISCUSSION TITLE
How to get last deployed time of a project?

### DISCUSSION THREAD

**Discussion by MaheshtheDev** [vercel/community#587] [Help] [answered]
I want to get the last deployed time. So that i can show in the website like when was the last time site was updated. If there is any api to get done exactly to fetch the last deployment details.

**Comment by leerob** [comment, marked as answer]
```js
// https://vercel.com/docs/rest-api#endpoints/deployments/get-a-deployment-by-id-or-url
const deploymentId = 'dpl_8ZQNkgXXt9V4vNf8kQTSLQhDdAr6' // replace with your own
// https://vercel.com/support/articles/how-do-i-use-a-vercel-api-access-token
const accessToken = process.env.VERCEL_ACCESS_TOKEN
const result = await fetch(
    `https://api.vercel.com/v13/deployments/${deploymentId}`,
    {
        method: 'GET',
        headers: {
            Authorization: `Bearer ${accessToken}`,
        }
    }
);

// ms since epoch for when the deployment finished
const { ready } = await result.json() // 1650903484801
// convert to human-readable date
const lastDeployedTime = new Date(ready).toLocaleString() // 4/25/2022, 9:18:04 AM
```

**Reply by MaheshtheDev** [reply]
How to get deploymentID?

**Reply by leerob** [reply]
Good point! `/v6/deployments` -> https://vercel.com/docs/rest-api#endpoints/deployments/list-deployments

**Reply by MaheshtheDev** [reply]
Is there any way, that i can get only particular project related deployments only, instead of fetching first 20 deployments of all projects

**Reply by leerob** [reply]
You can `.filter()` the list of deployments based on the project name or ID!

**Reply by xcarpentier** [reply]
Hi @leerob,
Not possible to use `process.env.VERCEL_DEPLOYMENT_ID` ?

**Comment by Pathsis** [comment]


**Reply by MaheshtheDev** [reply]
Yeah! you can use this in your api call and get the expected results

https://github.com/orgs/vercel/discussions/587#discussioncomment-2670125
